packages/shared-base/src/data/queryClient.ts

```diff
@@ -5,7 +5,6 @@ export const queryClient = new QueryClient({
         queries: {
             staleTime: 20_000,
             cacheTime: 300_000,
-            refetchOnWindowFocus: true,
```

Comment:
you can only disable it in the development mode.

Reply:
This option is true by default, so I just remove it.

Reply:
this option improves the user experience